src/employees/entities/employee_saving.entity.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  Entity,
+  Column,
+  PrimaryGeneratedColumn,
+  Generated,
+  CreateDateColumn,
+  UpdateDateColumn,
+  ManyToOne,
+} from 'typeorm';
+import { Employees } from './employees.entity';
+
+enum TypeSaving {
+  SIMPANAN = 'SIMPANAN',
+  AMBILAN = 'AMBILAN',
+}
+
+@Entity()
+export class Employee_Saving {
+  @PrimaryGeneratedColumn('uuid')
+  @Generated('uuid')
+  id_saving: string;
+
+  @Column()
+  tanggal: Date;
+
+  @Column({
+    type: 'enum',
+    enum: TypeSaving,
+    default: TypeSaving.SIMPANAN,
+  })
+  type: TypeSaving;
+
+  @Column()
+  total: number;
+
+  @Column({ type: 'text' })
+  description: string;
+
+  @ManyToOne(() => Employees, (employee) => employee.savings)
+  employee: Employees;
```

Comment:
use lazy relation
https://orkhan.gitbook.io/typeorm/docs/eager-and-lazy-relations

Reply:
done